src/api/nodeRpcCall.js

```diff
@@ -1298,6 +1298,35 @@ const getBlockEvents = async (blockHash) => {
   }
 }
 
+const getCongressCandidates = async () => {
+  const api = await getApi();
+  const electionsCandidates = await api.query.elections.candidates()
+  return electionsCandidates.toHuman()
```

Comment:
Not a blocker, but in the future only use `.toHuman()` when displaying stuff to a human. Definitely not in sagas/nodeRpcCall.js.